2-mandatory/2-water-bottle.js

```diff
@@ -18,19 +18,37 @@ You have to implement the missing features according to the specification.
 let bottle = {
   volume: 0,
   fillUp: function () {
-    // calling this function should completely fill your bottle (volume = 100);
+    if (this.volume == 0) {
+      this.volume = 100;
+    }
   },
   pour: function () {
-    // calling this function should increase your bottle volume by 10 units;
+    if (this.volume < 100) {
+      this.volume = this.volume + 10;
+    } else {
+      return this.volume;
+    }
   },
   drink: function () {
-    // calling this function should decrease your bottle volume by 10 units;
+    if (this.volume >= 10) {
+      this.volume = this.volume - 10;
+    } else {
+      return this.volume;
+    }
   },
   isFull: function () {
-    // this function should return true if your bottle is full;
+    if (this.volume == 100) {
+      return true;
+    } else {
+      return false;
+    }
   },
   isEmpty: function () {
-    // this function should return true if your bottle is empty;
+    if (this.volume == 0) {
+      return true;
+    } else {
+      return false;
+    }
```

Comment:
The best way is to use the Switch method here, so it will be shorter and easy to understand